src/deye_mqtt.py

```diff
@@ -35,48 +35,48 @@ def __init__(self, message: str):
 class DeyeMqttClient:
     def __init__(self, config: DeyeConfig):
         self.__log = logging.getLogger(DeyeMqttClient.__name__)
-        self.__mqtt_client = paho.Client(
+        self._mqtt_client = paho.Client(
```

Comment:
Please restore the original name. In this project private fields are prefixed with double underscore.

Reply:
I like to re-use the implementation in `MqttAutoConfPublisher` as it needs to publish the configuration for sensors via mqtt